bugwarrior/services/gitea.py

```diff
@@ -29,16 +29,28 @@
 
 log = logging.getLogger(__name__)  # pylint: disable-msg=C0103
 
+#TODO: Document this with docstrings
 class GiteaConfig(config.ServiceConfig):
     service: typing_extensions.Literal['gitea']
-
     host = "gitea.com"
-    login: str
     token: str
     username: str
-    password: str
-    exclude_repos = config.ConfigList([])
-    include_repos = config.ConfigList([])
+    include_assigned_issues: bool = False
+    include_created_issues: bool = False
+    include_mentioned_issues: bool = False
+    include_review_requested_issues: bool = False
```

Comment:
Rather than adding a separate configuration option for these specific purposes, I would rather see a more powerful `query` configuration option which can pass arbitrary filters to the gitea api. (This would apply to the `issue_limit` option as well.) See [github.query](https://bugwarrior.readthedocs.io/en/latest/services/github.html#queries) for example.